controllers/contactsController.js

```diff
@@ -36,7 +44,9 @@ const deleteContact = async (request, response, next) => {
 
 const updateContact = async (request, response, next) => {
     const { contactId } = request.params;
-   
+    // поверне стару версію, а в базі документ оновить
```

Comment:
Wszystkie komentarze raczej do usunięcia, a jeśli naprawdę uważsz, że powinny być to po Angielsku.